src/lib/createPollBlocks.ts

```diff
@@ -101,32 +111,33 @@ export function createPollBlocks(block: BlockBuilder, question: string, options:
     // Next Poll Button if live poll
     if (poll.pollIndex !== undefined && poll.totalLivePolls && (poll.pollIndex < poll.totalLivePolls - 1) && poll.activeLivePoll) {
         block
-        .addActionsBlock({
-            elements: [
-                block.newButtonElement({
-                    actionId: 'nextPoll',
-                    text: block.newPlainTextObject('Next Poll'),
-                }),
-            ],
-        });
+            .addActionsBlock({
+                elements: [
+                    block.newButtonElement({
+                        actionId: 'nextPoll',
+                        text: block.newPlainTextObject('Next Poll'),
+                    }),
+                ],
+            });
     }
 
     // Add Option button
     if (!poll.finished && poll.allowAddingOptions) {
         block
-        .addActionsBlock({
-            elements: [
-                block.newButtonElement({
-                    actionId: 'addUserChoice',
-                    text: block.newPlainTextObject('Add option'),
-                }),
-            ],
-        });
+            .addActionsBlock({
+                elements: [
+                    block.newButtonElement({
+                        actionId: 'addUserChoice',
+                        text: block.newPlainTextObject('Add option'),
+                    }),
+                ],
+            });
```

Comment:
These looks like whitespace changes, can you please remove them?

Reply:
@murtaza98 sir  I made the requested changes please have a look.